### ISSUE TITLE
ListBox: options not focusable using keyboard

### ISSUE THREAD

**Issue by nico-richter** [primefaces/primereact#3672]
### Describe the bug

Entry of ListBox is not focusable using the tab key. 
If using filter only InputText is focusable.

### PrimeReact version

8.7.2

### React version

18.x

### Language

ALL

### Build / Runtime

Create React App (CRA)

### Browser(s)

Chrome 107

### Steps to reproduce the behavior

1. Click/Focus on element before ListBox.
2. press Tab on keyboard

### Expected behavior

Behave like the native <select multiple> to focuses ListBox itself and let navigate with arrow keys. 
Another way is to focus the first or last selected option or the first option.

Focus via ref should also be possible.

**Comment by melloware** [comment]
@nico-richter all Accessibility and ARIA concerns are being addressed for 9.0.

**Comment by melloware** [comment]
This is how it will eventually work just like PrimeVue: https://www.primefaces.org/primevue/listbox
